.github/workflows/pullrequests.yml

```diff
@@ -61,8 +61,8 @@ jobs:
       - name: Run tests & collect coverage
         run: npm run test:ci
         env:
-          TMP_CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-          TMP_CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          TMP_CLOUDFLARE_API_TOKEN: ${{ secrets.NEW_CLOUDFLARE_API_TOKEN }}
```

Comment:
`env_var_v2_final_(1).docx`

Any way we can just re-use the old name? Fine to break the prerelease registry for a moment while getting this out if needed.

Reply:
Absolutely—just didn't want to break things. I suppose it's okay to break the pre-release registry for a bit, so I can rename them just before merging